### ISSUE TITLE
NmzCone: Vertex has non-positive denominator!

### ISSUE THREAD

**Issue by mkoeppe** [Normaliz/PyNormaliz#37]
```
PyNormaliz.NmzCone(cone=[], number_field=['a^2 - 2', 'a', '[1.414213562373095 +/- 2.99e-16]'], subspace=[], vertices=[[1L, 1L], [[[0L, 1L], [1L, 1L]], 1L]])
error: Some error in the normaliz input data detected: Vertex has non-positive denominator!
```

**Comment by sebasguts** [comment]
Actually, each number needs to be a list atm. I will fix this.

**Comment by sebasguts** [comment]
Fixed in 275a406bf24479da4c0af24d8c0f9072b7c8794c
